benchmark/sigma_v2/python/tools/analysis.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import os
+import re
+
+from counter_registry import CounterRegistry
+from model_config import ModelConfig
+
+gpu_flops_map = {
```

Comment:
just curious if using a fp8 model like Qwen/Qwen3-Coder-480B-A35B-Instruct-FP8 should this use fp8 flops?